I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 288, in script
    $ [name_p] = "Professor"
  File "game/script.rpy", line 288, in <module>
    $ [name_p] = "Professor"
ValueError: too many values to unpack (expected 1)

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 288, in script
    $ [name_p] = "Professor"
  File "C:\Users\Erik\renpy-8.3.2-sdk\renpy\ast.py", line 821, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\Erik\renpy-8.3.2-sdk\renpy\python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 288, in <module>
    $ [name_p] = "Professor"
ValueError: too many values to unpack (expected 1)

Windows-10-10.0.19045 AMD64
Ren'Py 8.3.2.24090902
Heapsort 1.0
Wed Dec  4 19:01:13 2024
